src/routing/settings-router/settings-router.tsx

```diff
@@ -19,10 +18,10 @@ const SettingsRouter: React.FC = () => (
       <AppVersion className={`${BLOCK_NAME}__app-version`} />
     </div>
     <div className={`${BLOCK_NAME}__data`}>
-      <Switch>
+      <Routes>
+        <Route path="" element={<Navigate to="profile" />} />
         {mapRoutes(routes)}
-        <Redirect to={PROFILE_SETTINGS_PATH} />
```

Comment:
In case if we go to page [](https://localhost:3000/settings/wrongPage)
![image](https://user-images.githubusercontent.com/52489178/188728748-bcca556c-cf30-48cf-bbb6-ca5867ae05da.png)
We remain on the unknown UI, we have either to redirect to PROFILE_SETTINGS_PATH or to 404 page